poetry.lock

```diff
@@ -2481,6 +2481,7 @@ numpy = [
     {version = ">=1.17.3", markers = "(platform_machine != \"aarch64\" and platform_machine != \"arm64\") and python_version < \"3.10\""},
     {version = ">=1.19.2", markers = "platform_machine == \"aarch64\" and python_version < \"3.10\""},
     {version = ">=1.20.0", markers = "platform_machine == \"arm64\" and python_version < \"3.10\""},
+    {version = ">=1.21.0", markers = "python_version >= \"3.10\""},
```

Comment:
This line looks wrong to me, but doesn't seem to cause a regression on Ubuntu 20 (Python 3.8)